homeassistant/components/yale_smart_alarm/alarm_control_panel.py

```diff
@@ -86,11 +85,13 @@ async def async_setup_entry(
 class YaleAlarmDevice(CoordinatorEntity, AlarmControlPanelEntity):
     """Represent a Yale Smart Alarm."""
 
-    coordinator: YaleDataUpdateCoordinator
-
-    _attr_name: str = coordinator.entry.data[CONF_NAME]
-    _attr_unique_id: str = coordinator.entry.entry_id
-    _identifier: str = coordinator.entry.data[CONF_USERNAME]
+    def __init__(self, coordinator: YaleDataUpdateCoordinator) -> None:
+        """Initialize the Yale Alarm Device."""
+        self.coordinator = coordinator
+        super().__init__(self.coordinator)
+        self._attr_name: str = self.coordinator.entry.data[CONF_NAME]
+        self._attr_unique_id: str = self.coordinator.entry.entry_id
```

Comment:
The entry_id will always be a string. We shouldn't need the explicit type here, unless the ConfigEntry typing isn't complete and we should improve it there.